src/kudu/consensus/raft_consensus.cc

```diff
@@ -2114,7 +2125,8 @@ Status RaftConsensus::RequestVote(const VoteRequestPB* request,
                                    << request->candidate_uuid()
                                    << " " << hostname_port
                                    << " for testing.";
-    return RequestVoteRespondVoteWitheld(request, hostname_port, response);
+    return RequestVoteRespondVoteWitheld(request, hostname_port,
+        "votes are being witheld for testing", response);
```

Comment:
Change not required?

Reply:
we require this because we changed how RequestVoteRespondVoteWitheld is implemented. It caters to 2 types of errors now.